Resources/Prototypes/Damage/modifier_sets.yml

```diff
@@ -173,6 +173,18 @@
     Cellular: 0.0
     Caustic: 0.0
 
+- type: damageModifierSet
+  id: SlimePet # Very survivable slimes, you should never put such high damage
```

Comment:
"you should never put such high damage"?

Reply:
corrected